sdk/formrecognizer/azure-ai-formrecognizer/tests/perfstress_tests/perf_dmac_build_model_requests.py

```diff
@@ -0,0 +1,41 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import os
+from azure_devtools.perfstress_tests import PerfStressTest
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.formrecognizer import DocumentModelAdministrationClient
+from azure.ai.formrecognizer.aio import DocumentModelAdministrationClient as AsyncDocumentModelAdministrationClient
+
+class BuildModelRequestPreparation(PerfStressTest):   
```

Comment:
For DMAC I can add perf tests for the other operations as well, let me know what you think.

Reply:
I'm good with this. For response testing, are you thinking of testing a get/list model op?

Reply:
That's a great idea, I'll do get model in my next PR to have some response handling coverage!